core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -92,6 +92,13 @@ private TableProperties() {}
   public static final String METADATA_COMPRESSION = "write.metadata.compression-codec";
   public static final String METADATA_COMPRESSION_DEFAULT = "none";
 
+  public static final String PREVIOUS_METADATA_LOG_MAX_COUNT = "write.metadata.previous-log-max-count";
```

Comment:
How about `write.metadata.previous-versions-max`? No need to refer to these as a log.

Reply:
updated